README.md

```diff
@@ -85,3 +85,24 @@ Peer requires a few things to run properly on different platforms. This list may
 - Tier 3 (We don't have the hardware to test but we'll do our best!)
   - win-arm64
   - osx-arm64
+
+## Environment variables
+
+All of peer configuration can be done either in an environment variable or within the config file. Nesting is done in environment variables with double underscores. So to set WatchIntervalSeconds variable you can simply do 
+
+in sh-like terminals
+
+```sh
+export PEER__WATCHINTERVALSECONDS=30
+```
+
+or in powershell/pwsh
+
+```powershell
+$env:PEER__WATCHINTERVALSECONDS=30 
+```
```

Comment:
```suggestion
Peer configuration can be specified in the config file or in environment variables. Environment variables take precedence over the config file. When using environment variables, the levels of nesting are represented using double underscores. For example, the environment variable `PEER__WATCHINTERVALSECONDS` would override the `Peer.WatchInvervalSeconds` value in the config file.

Setting a config variable in sh-like shells
\```
export PEER__WATCHINTERVALSECONDS=30
\```


Setting a config variable in pwsh/powershell
\```
$env:PEER__WATCHINTERVALSECONDS = 20
\```

```

Reply:
TBH I would leave out the snippets though. I like having them in walk through style stuff so I can copy/paste but this section is about using env vars VS the specific var so there's a good chance it's not the var someone wants to set. No issue with it being in though, just not something I would add myself.